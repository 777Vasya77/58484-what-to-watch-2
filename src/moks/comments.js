export const COMMENTS = [
  {
    'id': 1,
    'user': {
      'id': 18,
      'name': `Sophie`
    },
    'rating': 2,
    'comment': `Poised to be an instant classic, almost everything about this film is phenomenal - the acting, the cinematography, the discography, etc.`,
    'date': `2019-10-18T19:05:06.212Z`
  },
  {
    'id': 2,
    'user': {
      'id': 16,
      'name': `Mollie`
    },
    'rating': 2,
    'comment': `This movie really touched my heart, it really is the best movie of the year and everyone should see this masterpiece.`,
    'date': `2019-10-29T19:05:06.212Z`
  },
  {
    'id': 3,
    'user': {
      'id': 14,
      'name': `Corey`
    },
    'rating': 5,
    'comment': `A movie that will take you to another world full of emotions.`,
    'date': `2019-10-02T19:05:06.212Z`
  },
  {
    'id': 4,
    'user': {
      'id': 18,
      'name': `Sophie`
    },
    'rating': 4,
    'comment': `This movie is perfect in all its categories: credits, sound track, production, casting, writing, photography, editing, acting, and direction.\nI was amazed with the freedom of the use of the camera. This movie will change the way movies are made. Slow-mo, stills, black and white, and color were all used to brilliant effect.`,
    'date': `2019-10-27T19:05:06.212Z`
  },
  {
    'id': 5,
    'user': {
      'id': 18,
      'name': `Sophie`
    },
    'rating': 5,
    'comment': `I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.`,
    'date': `2019-10-20T19:05:06.212Z`
  },
  {
    'id': 6,
    'user': {
      'id': 14,
      'name': `Corey`
    },
    'rating': 5,
    'comment': `I personally found this movie to be boring. Definitely one of the most boring movies I've ever seen.`,
    'date': `2019-10-20T19:05:06.212Z`
  },
  {
    'id': 7,
    'user': {
      'id': 13,
      'name': `Zak`
    },
    'rating': 3,
    'comment': `Unfortunately we don't have a reliable way to tell the true ratings of a movie.`,
    'date': `2019-10-26T19:05:06.212Z`
  },

];
