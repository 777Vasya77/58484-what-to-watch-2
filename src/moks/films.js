export const FILMS = [
  {
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    image: `fantastic-beasts-the-crimes-of-grindelwald.jpg`
  },
  {
    title: `Aviator`,
    image: `aviator.jpg`
  },
  {
    title: `Bohemian Rhapsody`,
    image: `bohemian-rhapsody.jpg`
  },
  {
    title: `Macbeth`,
    image: `macbeth.jpg`
  },
  {
    title: `Mindhunter`,
    image: `mindhunter.jpg`
  },
  {
    title: `War of the Worlds`,
    image: `war-of-the-worlds.jpg`
  },
  {
    title: `Johnny English`,
    image: `johnny-english.jpg`
  },
  {
    title: `We Need to Talk about Kevin`,
    image: `we-need-to-talk-about-kevin.jpg`
  },
];
