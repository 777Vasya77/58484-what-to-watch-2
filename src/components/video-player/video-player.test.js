import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from '~/components/video-player/video-player';
import {FILMS} from '~/moks/test-moks';
import Film from '~/models/film';

const movie = new Film(FILMS[0]);

describe(`VideoPlayer component tests`, () => {
  it(`Component render correctly`, () => {
    const {previewVideoLink, previewImage} = movie;
    const tree = renderer.create(
        <VideoPlayer
          src={previewVideoLink}
          isPlaying={false}
          previewImage={previewImage}
        />,
        {
          createNodeMock: () => ({
            load: () => ({})
          })
        }
    );

    expect(tree).toMatchSnapshot();
  });
});
