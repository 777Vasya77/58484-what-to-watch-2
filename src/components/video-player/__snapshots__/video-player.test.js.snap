// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer component tests Component render correctly 1`] = `
<video
  className="player__video"
  height="175"
  muted={true}
  poster="https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  width="280"
/>
`;
